src/modules/tools/zprobe/CartGridStrategy.cpp

```diff
@@ -15,7 +15,8 @@
 
     The size of the grid can be set with...
 
-      leveling-strategy.rectangular-grid.size        7
+      leveling-strategy.rectangular-grid.grid_x_size        7
+      leveling-strategy.rectangular-grid.grid_y_size        7
```

Comment:
There is a note about "leveling-strategy.rectangular-grid.size"

 The size of the grid can be set with...
      leveling-strategy.rectangular-grid.size        7
    or
      leveling-strategy.rectangular-grid.grid_x_size        7
      leveling-strategy.rectangular-grid.grid_y_size        7
    this is the X and Y size of the grid, it must be an odd number, the default is 7 which is 49 probe points